vertex-ai/managed_notebook/README.md

```diff
@@ -0,0 +1,61 @@
+## Vertex AI Workbench with Java Client Library
+
+This is a small sample to create a managed notebook using the Java SDK
+
+### Useful information
+
+Before compiling, open
+java-docs-samples/vertex-ai/managed_notebook/src/main/java/com/gcp_aiml/samples/CreateRuntime.java and update the parameters at the top of the `public class CreateRuntime`.  You will need to know these values:
+
+Project ID
+Project Number
+
+You can find these values in the cloud console:
+
+![in the cloud console](img/project-view.png).
+
+In order to set a default load balancer provider, this maven package has a file
+called io.grpc.LoadBalancerProvider in the java-docs-samples/vertex-ai/managed_notebook/src/main/resources/META-INF/services
+directory with the value of $(eval echo
+"io.grpc.internal.PickFirstLoadBalancerProvider")
+
+- https://stackoverflow.com/questions/55484043/how-to-fix-could-not-find-policy-pick-first-with-google-tts-java-client
```

Comment:
Perhaps you would want to explain a bit more what the problem is here, and why you picked this solution. (and why they might choose another one in other circumstances)